src/portal/src/i18n/lang/ja-jp-lang.json

```diff
@@ -0,0 +1,1596 @@
+{
+    "APP_TITLE": {
+        "VMW_HARBOR": "Harbor",
+        "HARBOR": "Harbor",
+        "VIC": "vSphere Integrated Containers",
+        "MGMT": "Management",
+        "REG": "Registry",
+        "HARBOR_SWAGGER": "Harbor Swagger",
+        "THEME_DARK_TEXT": "ダークモード",
+        "THEME_LIGHT_TEXT": "ライトモード"
+    },
+    "SIGN_IN": {
+        "REMEMBER": "ログイン状態を記憶",
+        "INVALID_MSG": "ユーザー名またはパスワードが間違っています。",
+        "FORGOT_PWD": "パスワードを忘れた",
+        "HEADER_LINK": "サインイン",
+        "CORE_SERVICE_NOT_AVAILABLE": "コアサービスが利用不可になっています。"
+    },
+    "SIGN_UP": {
+        "TITLE": "登録"
+    },
+    "BUTTON": {
+        "STOP": "停止",
+        "CANCEL": "キャンセル",
+        "OK": "OK",
+        "DELETE": "削除",
+        "LOG_IN": "ログイン",
+        "LOG_IN_OIDC": "OIDCプロバイダ経由でログイン",
```

Comment:
Would it be possible to add a space between English character and Japanese character to align with VMware products?